teps/0048-task-results-without-results.md

```diff
@@ -670,6 +707,103 @@ Having default results defined in check-name-matches Task will fix the Pipeline.
 
 ## Alternatives
 
+### Specifying Default Value during Runtime
```

Comment:
why was this alternative rejected?

Reply:
I'm not sure if was explicitly rejected - I don't see it as orthogonal to the chosen one - but perhaps it would be best to clarify whether these alternatives are meant as not going to be implemented or possibly going to be implemented later?